parlai/tasks/vqa_v1/build.py

```diff
@@ -11,10 +11,13 @@
 
 def buildImage(opt):
     dpath = os.path.join(opt['datapath'], 'COCO-IMG')
+    version = '1'
 
-    if not build_data.built(dpath, version_string='1'):
+    if not build_data.built(dpath, version_string=version):
         print('[building image data: ' + dpath + ']')
-        build_data.remove_dir(dpath)
+        # if already built but updating version, remove outdated files
+        if build_data.built(dpath):
+            build_data.remove_dir(dpath)
         build_data.make_dir(dpath)
```

Comment:
yeah I think let's leave for now so it's clear what steps it's doing, people will just copy & paste it anyways